contracts/libraries/SponsoredCCTPQuoteLib.sol

```diff
@@ -38,7 +38,7 @@ library SponsoredCCTPQuoteLib {
     uint256 private constant HOOK_DATA_INDEX = 228;
 
     // Minimum length of the message body (can be longer due to variable actionData)
-    uint256 private constant MIN_MSG_BYTES_LENGTH = 664;
+    uint256 private constant MIN_MSG_BYTES_LENGTH = 696;
```

Comment:
adding 2 words to hook data should increase minimum length of the message body by 64 bytes